Extension/src/LanguageServer/ui_new.ts

```diff
@@ -109,7 +110,7 @@ export class NewUI implements UI {
 
         this.browseEngineStatusBarItem = vscode.languages.createLanguageStatusItem(`cpptools.status.${LanguageStatusPriority.Mid}.tagparser`, documentSelector);
         this.browseEngineStatusBarItem.name = localize("cpptools.status.tagparser", "C/C++ Tag Parser Status");
-        this.browseEngineStatusBarItem.detail = localize("indexing.files.tooltip", "Indexing Workspace");
+        this.browseEngineStatusBarItem.detail = localize("cpptools.detail.tagparser", "Warming up...");
```

Comment:
warming up?  I think "indexing" is more descriptive.

Reply:
I don't understand.

Reply:
It's a placeholder while the UI waited for the first status message about the parser. I could change it to "Loading..." like we have for Configuration or something else

Reply:
Initializing will work